CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+and others are GIS users. All users, regardless of the profession or skill level,
+has the ability to contribute to PROJ.4. Here's a few suggestion on how:
+
+* Help PROJ.4-users that is less experienced than yourself.
+* Write a bug report
+* Request a new feature
+* Write documentation for your favorite map projection
+* Fix a bug
+* Implement a new feature
+
+In the following sections you can find some guidelines on how to contribute.
+As PROJ.4 is managed on GitHub most contributions require that you have a GitHub
+user. Familiarity with [issues](https://guides.github.com/features/issues/) and
+the [GitHub Flow](https://guides.github.com/introduction/flow/) is an advantage.
+
+## Help a fellow PROJ.4 user
+The main forum for support for PROJ.4 is the mailing list. You can subscribe to
+the mailing list [here](http://lists.maptools.org/mailman/listinfo/proj) and read
+the archive [here](http://lists.maptools.org/pipermail/proj/).
+
+If you have questions about the usage of PROJ.4 the mailing list is also the place to go.
+Please *do not* use the GitHub issue tracker as a support forum. Your question is much
+more likely to be answered on the mailing list, as many more people follow that than
+the issue tracker.
+
+## Adding bug reports
+
+Bug reports are handled in the [issue tracker](https://github.com/OSGeo/proj.4/issues)
+on PROJ.4's home on GitHub. Writing a good bug report is not easy. But fixing a poorly
+documented bug is not easy either, so please put in the effort it takes to create a
+thorough bug report.
+
+A good bug report includes at least:
+
+* A title that quickly explains the problem
+* A description of the problem and how it can be reproduced
+* Version of PROJ.4 being used
+* Version numbers of any other relevant software being used, e.g. operating system
+* A description of what already has been done to solve the problem
+
+The more information that is given up front, the more likely it is that a developer
+will find interest in solving the problem. You will probably get follow-up questions
+after submitting a bug report. Please answer them in a timely manner if you have an
+interest in getting the issue solved.
+
+Finally, please only submit bug reports that are actually related to PROJ.4. If the
+issue materializes in software that uses PROJ.4 it is likely a problem with that
+particular software. Make sure that it actually is a PROJ.4 problem before you submit
+an issue. If you can reproduce the problem only by using tools from PROJ.4 it is
+definitely a problem with PROJ.4.
+
+## Feature requests
+
+Got an idea for a new feature in PROJ.4? Submit a thorough description of the new
+feature in the [issue tracker](https://github.com/OSGeo/proj.4/issues). Please
+include any technical documents that can help the developer make the new feature a
+reality. An example of this could be a publicly available academic paper that
+describes a new projection.
+
```

Comment:
Also, including a numerical test case will make it much easier to verify that an implementation of your requested feature actually works as you expect.